client-src/elements/chromedash-gate-chip.js

```diff
@@ -144,9 +146,9 @@ class ChromedashGateChip extends LitElement {
     const stateName = GATE_STATE_TO_NAME[this.gate.state];
     const className = stateName.toLowerCase().replaceAll(' ', '_');
     const iconName = GATE_STATE_TO_ICON[this.gate.state];
-
     return html`
       <sl-button pill size="small" class=${className}
+        style="border:${this.gate.id == this.selectedGate ? '2px solid var(--dark-spot-color)' : nothing}"
```

Comment:
For non-selected gates, I think this would produce `style="border:"`, which is invalid CSS.  It probably gets harmlessly ignored, but it looks like an error.  I would prefer if you used a CSS class or Lit's styleMap or classMap or make the entire style attribute value be in the true part of the condition so that yields `style=""` or avoid generating the style attribute at all in that case.